### PR TITLE
OCPBUGSM-42704: Update docs for SNO deployments

### PR DESCRIPTION
**Pull request by lpettyjo**
OCPBUGSM-42704: Update docs for SNO deployments

Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGSM-42704

Link to docs preview:https://54410--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-local.html#local-storage-install_persistent-storage-local

QE review:
- [x] QE has approved this change.

Additional information:

PTAL: @gnufied @duanwei33

### CODE REVIEW

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
@lpettyjo  just a few comments to consider. Thanks.

File: modules/persistent-storage-local-install.adoc [hunk 1]
```diff
@@ -34,6 +34,17 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
...
 
+. Optional: Allow local storage to run on the management pool of CPUs in single-node deployment.
++
+Use the Local Storage Operator in single-node deployments and allow the use of CPUs that belong to the `managment` pool. Perform this step on single-node installations that use management workload partitioning.
++
+To allow Local Storage Operator to run on the management CPU pool, run following command:
++
+[source,terminal]
+----
+$ oc annotate project workload.openshift.io/allowed='management'
```
Comment (duanwei33) [inline modules/persistent-storage-local-install.adoc:45]: 1. Missing the name "openshift-local-storage"
2. Looks like "oc annotate project" doesn't work: 
```
$ oc annotate project openshift-local-storage openshift.io/node-selector=''
The Project "openshift-local-storage" is invalid: 
* metadata.namespace: Invalid value: "openshift-local-storage": field is immutable
* metadata.namespace: Forbidden: not allowed on this type

$ oc annotate project openshift-local-storage workload.openshift.io/allowed='management'
The Project "openshift-local-storage" is invalid: 
* metadata.namespace: Invalid value: "openshift-local-storage": field is immutable
* metadata.namespace: Forbidden: not allowed on this type
* metadata.annotations[workload.openshift.io/allowed]: Invalid value: "management": field is immutable, try updating the namespace
```

3. Please use "oc annotate namespace" instead. (Including another cmd not from this PR)
```
 $ oc annotate namespace openshift-local-storage openshift.io/node-selector=''
namespace/openshift-local-storage annotated

$ oc annotate namespace openshift-local-storage workload.openshift.io/allowed='management'
namespace/openshift-local-storage annotated
```

File: modules/persistent-storage-local-install.adoc [hunk 2]
```diff
@@ -34,6 +34,19 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
...
 
+. Optional: Allow local storage to run on the management pool of CPUs in single-node deployment.
++
+Use the Local Storage Operator in single-node deployments and allow the use of CPUs that belong to the `managment` pool. Perform this step on single-node installations that use management workload partitioning.
++
+To allow Local Storage Operator to run on the management CPU pool, run following commands:
++
+[source,terminal]
+----
+$ oc annotate namespace openshift-local-storage openshift.io/node-selector=''
```
Comment (duanwei33) [inline modules/persistent-storage-local-install.adoc:45]: @lpettyjo 
I mean it should replace the previous existed Line34, not need to add here. (Sorry not clear in my last comment.)

File: modules/persistent-storage-local-install.adoc [hunk 3]
```diff
@@ -34,6 +34,19 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
 $ oc annotate project openshift-local-storage openshift.io/node-selector=''
```
Comment (duanwei33) [inline modules/persistent-storage-local-install.adoc:34]:
```suggestion
$ oc annotate namespace openshift-local-storage openshift.io/node-selector=''
```

File: modules/persistent-storage-local-install.adoc (hunk 2 above)
Comment (lpettyjo) [reply modules/persistent-storage-local-install.adoc:45]: OK. Thanks! Fixed.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/persistent-storage-local-install.adoc (hunk 2 above)
Comment (duanwei33) [reply modules/persistent-storage-local-install.adoc:45]: Sorry but I still see this line present, it should be removed here.

**Comment by duanwei33** [comment]
@lpettyjo As mentioned here (https://github.com/openshift/openshift-docs/pull/54410/files#r1071720252), the Line45 should not be added, that is for infrastructure nodes, not SNO. Could you remove it? Others look good to me.
`$ oc annotate namespace openshift-local-storage openshift.io/node-selector=''
`

File: modules/persistent-storage-local-install.adoc (hunk 2 above)
Comment (lpettyjo) [reply modules/persistent-storage-local-install.adoc:45]: Fixed.

**Pull request merged**